docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+services:
```

Comment:
Please add a `version: "3.3"` as it won't run on mallard without it.

Reply:
done